CaloMC/src/CaloShowerStepFromStepPt_module.cc

```diff
@@ -185,11 +185,11 @@ namespace mu2e {
       //
       // --- These are hardcoded to make sure changes are intended and carefully considered ---
       //
-      procCodes_[11].insert(   {2,12,13,16,17,21,23,29,40,49,58} );    // electron
-      procCodes_[-11].insert(  {2,12,13,16,17,21,23,29,40,49,58} );    // positron
-      procCodes_[22].insert(   {2,12,13,16,17,21,23,29,40,49,58} ); // photon
-      procCodes_[2112].insert( {2,12,13,16,17,21,23,29,40,49,58,74} ); // neutron
-      procCodes_[2212].insert( {16,17,21,23,29,40,45,49,58} );   // proton
+      procCodes_[11].insert(   {2,12,13,16,17,21,23,29,40,49,58,59,74} );  // electron
```

Comment:
These proc codes should be in prolog or moved to ProcessCode as a function.

Reply:
In a ideal world, they should be parameters, but I doubt we'll ever change them in the future. So I'll put them in the function so that they don't get changed by mistake.